modules/configuration/src/main/java/org/apache/ignite/configuration/ConfigurationChanger.java

```diff
@@ -210,19 +212,42 @@ public void initialize(ConfigurationType storageType) {
         }
     }
 
-    /** Temporary until the IGNITE-14372 */
-    public CompletableFuture<?> changeX(
-        List<String> path,
+    /** */
+    public CompletableFuture<Void> change(
         ConfigurationSource source,
         ConfigurationStorage storage
```

Comment:
Why do we need this `storage` at all? Can we calculate the correct storage internally based only on passed source?

Reply:
Well, this is a temporary thing for node bootstrap. Maybe we will actually delete this parameter in the future.